pydy/tests/test_utils.py

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env python
 
+from sympy import Symbol
+
```

Comment:
This whole addition seems like you are adding classes that 1) could be namedtuples or 2) are likely unnecessary. What is the reason to add all of these?

If you go this route, there are an infinite number of class types that can be added. I could possibly imagine that a class called `Constant` or `Parameter` could possibly be useful, but it still seems like we'd be adding lots of frivolous classes here.

From the "symbolic side" all you need is a SymPy symbol for each model parameter/constant. Eventually you want to tie a numerical value to that, but be able to easily change that numerical value for different simulations and analyses. Write now our rhs functions are generated as "parameter value agnostic" and the user can provide them at the time of evaluation of the EoMs, so I don't really see how you will propagate these values to the user when they simply want to evaluate the EoMs.